integration-cli/docker_cli_prune_unix_test.go

```diff
@@ -90,3 +91,85 @@ func (s *DockerDaemonSuite) TestPruneImageDangling(c *check.C) {
 	c.Assert(err, checker.IsNil)
 	c.Assert(strings.TrimSpace(out), checker.Not(checker.Contains), id)
 }
+
+func (s *DockerSuite) TestPruneContainerUntil(c *check.C) {
+	out, _ := dockerCmd(c, "run", "-d", "busybox")
+	id1 := strings.TrimSpace(out)
+	c.Assert(waitExited(id1, 5*time.Second), checker.IsNil)
+
+	until := daemonUnixTime(c)
+
+	out, _ = dockerCmd(c, "run", "-d", "busybox")
+	id2 := strings.TrimSpace(out)
+	c.Assert(waitExited(id2, 5*time.Second), checker.IsNil)
+
+	out, _ = dockerCmd(c, "container", "prune", "--force", "--filter", "until="+until)
+	c.Assert(strings.TrimSpace(out), checker.Contains, id1)
+	c.Assert(strings.TrimSpace(out), checker.Not(checker.Contains), id2)
+
+	out, _ = dockerCmd(c, "ps", "-a", "-q", "--no-trunc")
+	c.Assert(strings.TrimSpace(out), checker.Not(checker.Contains), id1)
+	c.Assert(strings.TrimSpace(out), checker.Contains, id2)
+}
+
+func (s *DockerDaemonSuite) TestPruneImageUntil(c *check.C) {
+	s.d.StartWithBusybox(c)
+
+	out, _, err := s.d.BuildImageWithOut("test1",
+		`FROM busybox
+                 LABEL foo=bar`, true, "-q")
+	c.Assert(err, checker.IsNil)
+	id1 := strings.TrimSpace(out)
+
+	until := daemonUnixTime(c)
+
+	out, _, err = s.d.BuildImageWithOut("test2",
+		`FROM busybox
+                 LABEL bar=foo`, true, "-q")
+	c.Assert(err, checker.IsNil)
+	id2 := strings.TrimSpace(out)
+
+	out, err = s.d.Cmd("image", "prune", "-a", "--force", "--filter", "until="+until)
+	c.Assert(err, checker.IsNil)
+	c.Assert(strings.TrimSpace(out), checker.Contains, id1)
+	c.Assert(strings.TrimSpace(out), checker.Not(checker.Contains), id2)
+
+	out, err = s.d.Cmd("images", "-q", "--no-trunc")
+	c.Assert(err, checker.IsNil)
+	c.Assert(strings.TrimSpace(out), checker.Not(checker.Contains), id1)
+	c.Assert(strings.TrimSpace(out), checker.Contains, id2)
+}
+
+func (s *DockerDaemonSuite) TestPruneNetworkUntil(c *check.C) {
```

Comment:
@dnephin I removes two integration tests of `TestPruneNetworkUntil` and `TestPruneImageUntil`, and added unit test in `client/` directory. The `TestPruneContainerUntil` is left alone, think it we may want to keep one integration test for this one. But let me know if unit tests is preferred as well in this case.